src/test/java/com/epam/training/student_gregory_kubya/MainPageTest.java

```diff
@@ -44,6 +52,28 @@ public void checkSearchResultQuantity() {
 
     Assertions.assertThat(mainPage.getSearchResultQuantity())
         .as("Check search result quantity")
-        .isGreaterThan(500);
+        .isGreaterThan(300);
+  }
+
+  @Test
+  public void checkMultipleItemsInCart() {
+    List<ProductDTO> productList = new ArrayList<>();
+    List<String> productsReferences = Arrays.asList("3687009", "9006113", "3856253");
+    for (String productReference : productsReferences) {
+      mainPage.openProductPage(productReference);
+      productPage.addToCart();
+      productList.add(productPage.createProductObject());
+    }
+    cartPage.openCartPage();
+
+    Assertions.assertThat(cartPage.productNamesOnCart())
+        .as("Product not found in cart")
```

Comment:
the error message is a matter of taste. I prefer to explain what was performed like "Check all products are present on the cart page" and then there will be a message "Expected bla bla to contain... but was not found"